Docs: Documentation for 6.6 Explore and Logs panel features (#21754)
grafana/grafana: docs/sources/features/explore/index.md

- [InfluxDB](../datasources/influxdb)
- [Elasticsearch](../datasources/elasticsearch)

### Visualization Options

> 

Reviewer: Please add a paragraph after Deduping so that we avoid the stacked headings, per the Documentation Style Guide.

Doesn't have to be long, just something to help the reader understand what this section is about.

Author: ✅ Added

#### Deduping

Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:

* `exact` Exact matches are done on the whole line, except for date fields.
* `numbers` Matches on the line after stripping out numbers (durations, IP addresses etc.).
* `signature` The most aggressive deduping - strips all letters and numbers, and matches on the remaining whitespace and punctuation.

#### Time

> Shows or hides the time column. This is the timestamp associated with the log line as reported from the datasource.

Reviewer: Per the Documentation Style Guide, "data source" is two words.

Author: ✅ updated

> Shows or hides the time column. This is the timestamp associated with the log line as reported from the datasource.

Author: ```suggestion
Shows or hides the time column. This is the timestamp associated with the log line as reported from the data source.
```

#### Unique labels

Shows or hides the unique labels column that includes only non-common labels. All common labels are displayed above.

#### Wrap lines

> Wrap-lines option for logs, as for some of our users it’s more efficient to see one line per log message. The wrapped-line option is set as a default, unwrapped setting results in horizontal scrolling.

Reviewer: Do you mean that this option toggles line wrapping? I would say something like 

Set this to True if you want the display to use line wrapping. Otherwise, set it to False. (Or however you set this.)

- [InfluxDB](../datasources/influxdb)
- [Elasticsearch](../datasources/elasticsearch)

> ### Visualization Options

Reviewer: ```suggestion
### Visualization options
```

> It's possible to customize how logs are displayed and select which columns are shown.

Reviewer: ```suggestion
You can customize how logs are displayed and select which columns are shown.
```

* `exact` Exact matches are done on the whole line, except for date fields.
* `numbers` Matches on the line after stripping out numbers (durations, IP addresses etc.).
* `signature` The most aggressive deduping - strips all letters and numbers, and matches on the remaining whitespace and punctuation.

#### Time

Shows or hides the time column. This is the timestamp associated with the log line as reported from the data source.

#### Unique labels

Shows or hides the unique labels column that includes only non-common labels. All common labels are displayed above.

#### Wrap lines

Set this to True if you want the display to use line wrapping. If set to False, it will result in horizontal scrolling.

### Labels and Parsed fields

Each log row has an extendable area with its labels and parsed fields, for more robust interaction. For all labels we have added the ability to filter for (positive filter) and filter out (negative filter) selected labels. Each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.

> ### Loki-specific Features

Reviewer: ```suggestion
### Loki-specific features
```
